modules/drivers/sqlite/src/metabase/driver/sqlite.clj

```diff
@@ -109,9 +109,9 @@
 ;; The normal SELECT * FROM table WHERE 1 <> 1 LIMIT 0 query doesn't return any information for SQLite views -- it
 ;; seems to be the case that the query has to return at least one row
 (defmethod sql-jdbc.sync/fallback-metadata-query :sqlite
-  [driver schema table]
+  [driver db schema table]
   (sql.qp/format-honeysql driver {:select [:*]
-                                  :from   [[(h2x/identifier :table schema table)]]
+                                  :from   [[(h2x/identifier :table db schema table)]]
```

Comment:
This doesn't make logical sense to allow sqlite to have an identifier like `x.y.z`. See this comment for a solution: offer a 3-arity implementation instead https://github.com/metabase/metabase/pull/36121/files#r1410580202